docker/docker-compose.yml

```diff
@@ -1,12 +1,24 @@
 version: '3.4'
 
 services:
+  db:
+    image: postgres:14
+    environment:
+      POSTGRES_USER: username
+      POSTGRES_PASSWORD: password
```

Comment:
good to use env vars?

```yml
  POSTGRES_USER: ${DB_USER}
  POSTGRES_PASSWORD: ${DB_PASSWORD}
  ```

Reply:
wanna avoid having the overhead of the sh file for the db container and the env file would be pushed so will avoid for this, adding vars for the committer